core/src/message/stored.rs

```diff
@@ -76,6 +76,19 @@ impl StoredMessage {
         )
     }
 
+    /// Convert Stored Message into Message.
+    pub fn into_message(&self) -> Message {
```

Comment:
Do we really need this conversion? And usually `into_` methods consume `self` according to Rust naming convention.

https://rust-lang.github.io/api-guidelines/naming.html

Reply:
`StoredMessage` shouldn't conceptually be able to convert into `Message`. If you really need this conversion, you can manually call Message::new constructor